Examples/src/components/Examples/Charts2D/Filters/PercentageChange/index.tsx

```diff
@@ -241,7 +60,23 @@ export default function PercentageChange() {
                         Original Data
                     </ToggleButton>
                 </ToggleButtonGroup>
-                <div id={divElementId} className={localClasses.chartArea} />
+                {/* // Usage in SciChartReact */}
+                <SciChartReact
+                    key={chartKey} // Change the key to force re-render
+                    initChart={(rootElement) => drawExample(rootElement, usePercentage)}
+                    className={classes.ChartWrapper}
+                    onInit={(initResult: { sciChartSurface: SciChartSurface; wasmContext: any }) => {
+                        const { sciChartSurface } = initResult;
+                        sciChartSurfaceRef.current = sciChartSurface;
+
+                        // Cleanup on unmount
+                        return () => {
```

Comment:
This is unnecessary. As SciChartReact should already do this anyway

Reply:
FYI, desctructor callback is now a feature in SciChartReact. But here it is still unnecessary